source/reference/config/triggers.txt

```diff
@@ -117,10 +150,55 @@ trigger configuration files:
        "match": { <Match Filter> },
        "project": { <Projection Filter> },
      },
-     "function_name": "<Trigger Function Name>",
-     "disabled": <Boolean>
+     "event_processors": {
+        "FUNCTION": {
+            "config": {
+                "function_name": "<Function Name>"  
+            }
+        }
+     }
    }
 
+.. _trigger-eventbridge-event-processor:
+
+**Database Trigger with an AWS EventBridge Event Processor**
+
+.. code-block:: json
```

Comment:
fyi - this isn't formatting correctly for some reason (no syntax highlighting or caption)

Reply:
I'm not sure why :(( I'll keep playing around with it

Reply:
Didn't work :( I'll keep trying it!